install-wings.sh

```diff
@@ -447,68 +394,83 @@ function firewall_ufw {
   ufw status numbered | sed '/v6/d'
 }
 
-function firewall_firewalld {
+firewall_firewalld() {
   echo -e "\n* Enabling firewall_cmd (firewalld)"
   echo "* Opening port 22 (SSH), 8080 (Daemon Port), 2022 (Daemon SFTP Port)"
 
   # Install
-  [ "$OS_VER_MAJOR" == "7" ] && yum -y -q update
   [ "$OS_VER_MAJOR" == "7" ] && yum -y -q install firewalld > /dev/null
-  [ "$OS_VER_MAJOR" == "8" ] && dnf -y -q update
   [ "$OS_VER_MAJOR" == "8" ] && dnf -y -q install firewalld > /dev/null
 
   # Enable
   systemctl --now enable firewalld > /dev/null # Enable and start
 
   # Configure
+  firewall-cmd --ad-service=ssh --permanent -q # Port 22
```

Comment:
Oops, this is a misspelling. I can see that you are not the one to blame since the invalid command is in the `master` branch as of right now! Has it been broken all this time? 😱
```suggestion
  firewall-cmd --add-service=ssh --permanent -q # Port 22
```